main.js

```diff
@@ -545,3 +557,26 @@ function getCurrency (currencyString, currencySign) {
 	  }
 	});
 }
+
+function ConvertCurrency (currencyString, currencySign1, currencySign2 ) {
+	var appid = "b9bd43152a7982d06499"; // got my own api code :O
+	var url = "https://free.currencyconverterapi.com/api/v6/convert?q="+currencySign1+"_"+currencySign2+"&compact=ultra&apiKey="+appid;
+	request(url, function (err, response, body) {
+	  if(err){
+	    console.log('error converting currency:', error);
+	  }
+	  else {
+	    console.log('currency converted:', body);
+	    var sample = JSON.parse(body);
+	    if(Object.getOwnPropertyNames(sample).length){
+		var currencyConv = sample[currencySign1+currencySign2];
```

Comment:
you need to make sure the attribute you're trying to access in the `sample` object matches what you're trying to get